### PR TITLE
#113 Flatlist not scrollable on iOS

### PR DESCRIPTION
**Pull request by pwysowski**
### Description

- Added example of how can nested flatlist be implemented when using StickyParallaxHeader
- Added onTopReached prop
- Added info about it to CUSTOM.md and README.md

### CODE REVIEW

File: example/src/constants/screenStyles.js [hunk 1]
```diff
@@ -1,20 +1,27 @@
-import { StyleSheet, Platform } from 'react-native'
-import { ifIphoneX } from '../constants/utils'
-import colors from './colors'
-import constants from './constants'
+import {StyleSheet, Platform} from 'react-native';
```
Comment (davidwico) [inline example/src/constants/screenStyles.js:1]: I don't know if it's intentional, but I checked the rest of the code and there are no semicolons there. It looks like a whole file was parsed by wrong Prettier settings.

File: example/src/screens/CardScreen/CardScreen.js [hunk 2]
```diff
@@ -12,9 +21,14 @@ class CardScreen extends React.Component {
     super(props)
     this.state = {
       headerLayout: {
-        height: 0
-      }
-    }
+        height: 0,
+      },
+
```
Comment (davidwico) [inline example/src/screens/CardScreen/CardScreen.js:26]: Unnecessary space detected :)

File: example/src/screens/CardScreen/CardScreen.js [hunk 3]
```diff
@@ -125,6 +139,52 @@ class CardScreen extends React.Component {
...
+          <QuizCard
+            data={item}
+            num={index}
+            key={item.question}
+            cardsAmount={100}
+          />
+        )}
+        onScroll={this.onScroll}
+        scrollEnabled={
+          Platform.OS === 'android' ? true : this.shouldBeEnabled()
```
Comment (davidwico) [inline example/src/screens/CardScreen/CardScreen.js:181]: Please use `isAndroid` property from constants
https://github.com/netguru/sticky-parallax-header/blob/9fdbb14afbcd0048bf9978f3e1b276144a790074/src/constants/constants.js#L19

File: example/src/screens/CardScreen/CardScreen.js [hunk 4]
```diff
@@ -149,8 +223,10 @@ class CardScreen extends React.Component {
           headerSize={this.setHeaderSize}
           headerHeight={sizes.cardScreenHeaderHeight}
           background={this.renderBackground(user)}
-        >
-          {this.renderContent(user)}
+          onEndReached={this.stickyHeaderEndReached}
+          onTopReached={this.stickyHeaderTopReached}>
+          {/* {this.renderContent(user)} */}
```
Comment (davidwico) [inline example/src/screens/CardScreen/CardScreen.js:228]: This comment is necessary?

File: example/src/constants/screenStyles.js (hunk 1 above)
Comment (pi0trpietruszewski) [reply example/src/constants/screenStyles.js:1]: done

File: example/src/screens/CardScreen/CardScreen.js (hunk 2 above)
Comment (pi0trpietruszewski) [reply example/src/screens/CardScreen/CardScreen.js]: space removed

File: example/src/screens/CardScreen/CardScreen.js (hunk 3 above)
Comment (pi0trpietruszewski) [reply example/src/screens/CardScreen/CardScreen.js:180]: replaced with isAndroid

File: example/src/screens/CardScreen/CardScreen.js (hunk 4 above)
Comment (pwysowski) [reply example/src/screens/CardScreen/CardScreen.js:226]: Commented out code can be uncommented by users who don't know how to handle nested flatlist. Maybe it should also be documented in readme file?

**Pull request merged**